chichiwang/exercise-js
# Rename `webpack-config/` dir to `webpack/`

## Description
Linked to Issue: #45 
Reorganize Webpack configurations directory to better position it for extension to allow production build configurations.

## Changes
* Change the `webpack-config/` directory name to `webpack/`
  * Shorter name for access via npm scripts
* Rename `webpack-config.js` to `config.js` and move it to `webpack/config.js`
* Update scripts in `package.json`
  * To reference the new webpack config file location: `webpack/config.js`
  * `build:dev:analyze`: remove redundant step `npm run clean`

## Steps to QA
* Pull down and switch to this branch
* Run `npm run watch` to ensure nothing broke
* Run `npm run build:dev` to ensure nothing broke
* Run `npm run build:dev:analyze` to ensure nothing broke
